Documentation/project-docs/benchmarking.md

```diff
@@ -1,239 +1,83 @@
-# Benchmarking .NET Core 2.0 / 2.1 applications
+# Benchmarking .NET Core applications
 
 We recommend using [BenchmarkDotNet](https://github.com/dotnet/BenchmarkDotNet) as it allows specifying custom SDK paths and measuring performance not just in-proc but also out-of-proc as a dedicated executable.
 
 ```
 <ItemGroup>
-   <PackageReference Include="BenchmarkDotNet" Version="0.10.13" />
+   <PackageReference Include="BenchmarkDotNet" Version="0.11.1" />
 </ItemGroup>
 ```
 
 ## Defining your benchmark
 
 See [BenchmarkDotNet](http://benchmarkdotnet.org/Guides/GettingStarted.htm) documentation -- minimally you need to adorn a public method with the `[Benchmark]` attribute but there are many other ways to customize what is done such as using parameter sets or setup/cleanup methods. Of course, you'll want to bracket just the relevant code in your benchmark, ensure there are sufficient iterations that you minimise noise, as well as leaving the machine otherwise idle while you measure.
```

Comment:
This link should be updated: `https://benchmarkdotnet.org/articles/guides/getting-started.html`
I added redirects for all "old-style" doc pages, so people will see this page anyway, but I still want to update links wherever possible.